EIPS/eip-1898.md

```diff
@@ -4,7 +4,7 @@ title: Add `blockHash` to defaultBlock methods
 description: Add `blockHash` option to JSON-RPC methods that currently support defaultBlock parameter.
 author: Charles Cooper (@charles-cooper)
 discussions-to: https://ethereum-magicians.org/t/eip-1898-add-blockhash-option-to-json-rpc-methods-that-currently-support-defaultblock-parameter/11757
-status: Last Call
+status: Final
```

Comment:
you need to remove last call deadline field below

Reply:
done